avro_schema/frontend.lua

```diff
@@ -389,7 +393,8 @@ copy_schema = function(schema, ns, scope, open_rec, options)
                 local symbolmap = {}
                 for symbolno, symbol in ipairs(xsymbols) do
                     symbol = tostring(symbol)
-                    if not validname(symbol) and not options.utf8_enums then
+                    if not validname(symbol)
+                        and not context.options.utf8_enums then
```

Comment:
Indent.

Reply:
what is wrong with the indent?

Reply:
Condition is one the same level as the branch body, so some visual separator is needed. Eigther double indent for condition (I ised this method personally) or `then` on separate line.